csi.proto

```diff
@@ -245,31 +245,25 @@ message VolumeInfo {
 
   // Contains identity information for the created volume. This field is
   // REQUIRED. The identity information will be used by the CO in
-  // subsequent calls to refer to the provisioned volume.
-  VolumeHandle handle = 2;
+  // subsequent calls to refer to the provisioned volume. This field
+  // should not exceed 1MiB.
+  string id = 2;
+
+  // Attributes reflect static properties of a volume and MUST be passed
+  // to volume validation and publishing calls. Attributes MAY NOT
+  // uniquely identify a volume. A volume uniquely identified by `id`
+  // SHALL always report the same attributes. This field is REQUIRED.
+  Attributes attributes = 3;
```

Comment:
Hi @jdef,

It doesn't make sense to me that `Attributes` should be a discreet message type when it could simply be defined like so:

```proto
map<string,string> attributes = 3;
```

I dislike the trend in this spec to wrap single values.

Reply:
+1

I don't have a strong opinion, but given that `parameters` is in `map<string, string>`, i'd prefer use do `map<string,string> attributes = 3;` here.

Reply:
FWIW, my opinion **is** strong. I prefer consistency :)

Reply:
+1